modules/ROOT/pages/keycloak/coding.adoc

```diff
@@ -9,74 +9,123 @@ This guide will help you to set up the {keycloak-service} service SDK in your Ap
 
 == Importing the libraries
 
 [role="primary"]
 .Android
 ****
-. Import Core module.
-
-. Specify the schema of the redirect url for your Android project in `android.defaultConfig.manifestPlaceholders` of the apps `build.gradle` file.
+. Import the auth module
 +
-NOTE: It should match the schema specified in the `Valid Redirect URIs` section for the client in Keycloak.
-
-. TODO: Import auth module.
+[source,groovy,subs="attributes"]
+----
+dependencies {
+    implementation "org.aerogear:aerogear-auth:{release-number}"
+}
+----
 ****
 
 [role="secondary"]
 .iOS
 ****
-* Add dependency to your Podfile:
+. Add the dependency to your Podfile
++
+[source,ruby,subs="attributes"]
+----
+target '[TARGET NAME]' do
+    pod 'AGSAuth', '{release-number}'
+end
+----
+
+. Update the dependencies
 +
+[source,bash]
 ----
-pod 'AGSAuth'
+pod install
+----
+
+. Import the SDK
++
+[source,swift]
+----
+import AGSAuth
 ----
 ****
 
 [role="secondary"]
 .Cordova
 ****
+Install the auth package:
+[source,bash]
+----
+npm install @aerogear/auth
+----
+****
 
+[role="secondary"]
+.Xamarin
+****
+. Install link:https://docs.microsoft.com/en-us/nuget/install-nuget-client-tools[NuGet]
+
+. Install the Core package:
++
+[source,bash,subs="attributes"]
+----
+dotnet add package AeroGear.Mobile.Core --version {release-number}
+----
+
+. For Android run:
++
+[source,bash,subs="attributes"]
+----
+dotnet add package AeroGear.Mobile.Core.Platform.Android --version {release-number}
+----
+
+. For iOS run:
++
+[source,bash,subs="attributes"]
+----
+dotnet add package AeroGear.Mobile.Core.Platform.iOS --version {release-number}
+----
 ****
 
 == Initializing the SDK
 
 [role="primary"]
 .Android
 ****
-`AuthService` can be retrieved using the `MobileCore#getInstance` method.
 
+. Retrieve the auth service
++
 [source,java]
 ----
 AuthService authService = MobileCore.getInstance().getService(AuthService.class);
 ----
 
-Any subsequent `getService` call will return the same instance of `AuthService`.
-
-Before the `AuthService` can be used `AuthService#init` must be invoked once in an app.
-
-[source,java]
+. Specify the redirect URL. It is recommended to use the package name of your app.
++
 ----
-// Specify redirect URI. It is recommended to use the package name of your app.
 AuthServiceConfiguration authServiceConfig = new AuthServiceConfiguration
     .AuthConfigurationBuilder()
     .withRedirectUri("org.aerogear.mobile.example:/callback")
     .build();
+----
 
-// You only need to invoke this once every subsequent retrieval of the AuthService
-// will retrieve the same instance.
+. Before the AuthService can be used `init` must be invoked once in an app
++
+[source,java]
+----
 authService.init(context, authServiceConfig);
 ----
 
-Follow xref:keycloak/android-additional-configuration.adoc[this guide] for additional configuration.
+. Follow xref:keycloak/android-additional-configuration.adoc[this guide] for additional configuration.
```

Comment:
Let's say explicitly what is to be followed: For additional configuration see [Additional configuration for Android](https://www.github.com/aerogear/mobile-docs/).

Reply:
This document will no longer be included in the 1.0.0-alpha release

Reply:
@pb82 : not sure if this can impact, but for Xamarin we have already 1.0.0-alpha-1 and we are going to have 1.0.0-alpha-2 soon (we just made important fixes into the SDK that should be released to be used by the showcase)